src/main/resources/aws/load-balancer.yaml

```diff
@@ -39,15 +39,10 @@ Resources:
             -
               Name: "CookieName"
               Value: "JSESSIONID"
-  HttpSecurityGroup:
+      Scheme: internal
```

Comment:
My final tests revealed that I forgot to remove this one line after changing the approach from fully internal communication to dynamicly assigned access.

With this `Scheme: internal` the load balancer is only accessable from internal network, which was intended when I was passing it's private IP to the VUs.

Problematic thing is that we don't have integration tests in this repo for this load balancer. We are using it internally though, so running those with bumped aws-infrastrusture dependency SNAPSHOT failed.

Unfortunately that means I will need to add at least one more commit and @mzyromski-atlassian is on holidays, so he won't be able to appove. I'm counting on other reviewers.